packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewer.js

```diff
@@ -19,7 +19,7 @@ const SingleImageViewer = React.forwardRef(function SingleImageViewer ({ onError
         onError={onError}
         onLoad={onLoad}
       />
-      <InteractionLayer />
+      <InteractionLayer svg={ref ? ref.current : null} />
```

Comment:
In Safari, this ref remains `null` after the image loads. When looking at the parent component, `SingleImageViewerContainer`, it appears that the `onLoad()` method never runs.

Reply:
See #1094.

Reply:
Should be fixed by #1236.

Reply:
Confirming that this issue is resolved by #1236.